cranelift/codegen/src/isle.rs

```diff
@@ -0,0 +1,22 @@
+// GENERATED BY ISLE. DO NOT EDIT!
```

Comment:
Is this file leftover from an earlier generated-code setup? I don't see where it's generated or used (but possibly I'm missing something)... and `src/clif.isle` is also used to generate the x64-specific lowering code so I don't think we need this separate machine-independent one?

Reply:
Whoops yeah this is unused.